docker-compose.yml

```diff
@@ -5,6 +5,8 @@ services:
     build:
       context: .
       dockerfile: ./apps/web/Dockerfile
+    container_name: web
+    image: web:transcendence
```

Comment:
Why is the name of the image `web` and the tag `transcendence`? Isn't it supposed to be the name of the project and the version (e.g. ft_transcendence:latest)?